backends-velox/src/test/scala/org/apache/gluten/execution/VeloxRoughCostModelSuite.scala

```diff
@@ -43,7 +43,6 @@ class VeloxRoughCostModelSuite extends VeloxWholeStageTransformerSuite {
   override protected def sparkConf: SparkConf = super.sparkConf
     .set(GlutenConfig.RAS_ENABLED.key, "true")
     .set(GlutenConfig.RAS_COST_MODEL.key, "rough")
-    .set(GlutenConfig.VANILLA_VECTORIZED_READERS_ENABLED.key, "false")
```

Comment:
Checked, adding this configuration has no effect on the results of the following cases. Because no matter
enable: FileScan(Vectorized) -> ColumnarToRow
disabled: FileScan(row)
their costs are the same, and have to add the r2c node later, if don't want fallback